src/formElementPlugins/SummaryViewWrapper.tsx

```diff
@@ -15,10 +15,18 @@ const SummaryViewWrapper: React.FC<SummaryViewWrapperProps> = (props) => {
 
   // TO-DO: Provide a Default SummaryView (just label/text) if SummaryView not provided in plugin
 
+  const PluginComponent = <SummaryView parameters={parameters} value={value} />
+
   return (
     <ErrorBoundary pluginCode={pluginCode}>
       <React.Suspense fallback="Loading Plugin">
-        <SummaryView parameters={parameters} value={value} />
+        {isRequired ? (
+          <Form.Field required inline>
+            {PluginComponent}
+          </Form.Field>
+        ) : (
+          <Form.Field inline>{PluginComponent}</Form.Field>
+        )}
```

Comment:
```suggestion
<Form.Field required={isRequired} inline>{PluginComponent}</Form.Field>
```
Same suggestion as other PR.

Reply:
Nice! I wasn't sure if I could do that. Thanks